### DISCUSSION TITLE
Get Command name when using the same command

### DISCUSSION THREAD

**Discussion by MattMski** [discord-php/DiscordPHP#1191] [Q&A] [answered]
Hello, I wanted to know if it was possible to get the command name while using actual command.

For example if someone uses !news, they will get !news [topic], with !news coming from something internal. Instead of me having to extract the message content

**Comment by SQKo** [comment]
Nope, you will have to extract it on your own

If your actual problem is getting only the parameters without the command name, I suggest using the message-based command client: 
https://github.com/discord-php/DiscordPHP/wiki/Command-Client

**Comment by MattMski** [comment]
Hey @SQKo I do use the message-based command client for my commands. 

I was just looking for a way to extract the command or a sub command, every time the command is called

**Comment by MattMski** [comment]
Hey @SQKo wanted to know if there was any update from your end

**Reply by SQKo** [reply, marked as answer]
As I've stated before, you have to extract it on your own

We don't have plan to work on adding this to the library as discord generally recommends use of slash commands instead of old message based commands, but if you wish so you can submit your Pull request
